### ISSUE TITLE
be sure the user to remain is not suspended

### ISSUE THREAD

**Issue by jpahullo** [ndunand/moodle-tool_mergeusers#162]
We have faced the problem that a whole cycle have merged user B into A, and then A into B, leaving both users as suspended.

We should ensure the user to remain is NOT suspended.

**Comment by jpahullo** [comment]
from the climerger, I can see:

```
$ docker exec -it moodle37_webserver_1 php admin/tool/mergeusers/cli/climerger.php
== Merge user accounts ==
Introduce pairs of user's id to merge the first one into the

second one. The first user id (fromid) will 'lose' all its data to be 'migrated'

into the second one (toid). The user 'toid' will include data from both users.

To stop merging, Ctrl+C or type -1 either on fromid or toid fields.

Source user id (fromid):
: 3
Target user id   (toid):
: 4
Success. Log id: 25

Source user id (fromid):
: 4
Target user id   (toid):
: 3
Success. Log id: 26

Source user id (fromid):
:
```

and then

![image](https://user-images.githubusercontent.com/2048296/63152144-66009c00-c00b-11e9-8f05-e2af9a1af0a3.png)

![image](https://user-images.githubusercontent.com/2048296/63152187-8597c480-c00b-11e9-99e5-058246aa8f0e.png)

**Comment by jpahullo** [comment]
this is addressed by PR #163

**Comment by jpahullo** [comment]
done
